Yoast/yoastseo-amp
# Sunset the plugin

## Summary

This PR can be summarized in the following changelog entry:

* Prepares the plugin for dismissal.

## Relevant technical choices:

- bumped version to 0.7
- added a WP admin notice of type "info", see screenshot
- for now, the admin notice appears on the following admin pages: 'plugins.php', 'plugin-install.php', 'update-core.php'
- removed installation instructions
- removed screenshots
- the above changes will be visible also on the WP plugins directory at https://wordpress.org/plugins/glue-for-yoast-seo-amp/
- removed contributing instructions and the contributing.md file 
- kept the issue and PR templates: please do let me know whether they should be removed
- kept the banner and icon images because I'm assuming this last release will be published on the WP plugins repo anyways: please do let me know whether they should be removed
- kept the license.txt file: please do let me know if it should be removed
- kept `.gitattributes` and `.gitignore` files: please do let me know whether they should be removed
- removed stuff related to travis, codeclimate, phpcs, jscsrc, composer, etc.


## Test instructions

This PR can be tested by following these steps:

- switch to this branch
- activate the plugin
- see the info notice appears on the 'plugins.php', 'plugin-install.php', 'update-core.php' pages
- check there are no errors
- check the `readme.txt` and `README.md` files content

Fixes https://github.com/Yoast/yoastseo-amp/issues/141
